scripts/fields_header.yml

```diff
@@ -0,0 +1,6 @@
+---
+- key: ecs
+  title: ECS
+  description: >
+    ECS fields.
+  fields:
```

Comment:
Not familiar with the automation we have in Beats that will consume this. I won't nest everything under `ecs.*`, will it?

Reply:
It will not. There is quite a bit of history on why this additional thingy is here but in short it has to do with docs and how the fields.yml was organised in the past.